include/ddc/kernels/splines/view.hpp

```diff
@@ -92,6 +92,18 @@ namespace ddc {
 template <std::size_t N, class ElementType>
 using SpanND = std::experimental::mdspan<ElementType, std::experimental::dextents<std::size_t, N>>;
 
+template <std::size_t N, class ElementType>
+using SpanND_left = std::experimental::mdspan<
```

Comment:
see if this file can be largely simplified (or even removed ?)

Reply:
Actually I am not sure what I should keep and remove in this file. I let you say if you want me to restore chunks of codes.